### ISSUE TITLE
Formatting switch case body break indentation [0.47.2]

### ISSUE THREAD

**Issue by Panajev** [nicklockwood/SwiftFormat#792]
Hello @nicklockwood ,

Not sure if this is a known issue or a regression, but essentially this code is transformed from:
```swift
        switch config.option {
            case .text: do {
                let biometricButton = BiometricWithText(frame: .zero)
                biometricButton.textLabel.text = NativeLobbyLoginModule.titleForBiometricAuth
                view.addSubview(biometricButton)
                view.addSubviewToEdges(biometricButton)
                biometricButton.addTarget(self,
                                      action: #selector(self.performFingerprintLogin),
                                      for: .touchUpInside)
            }        
```

to

```swift
            case .text: do {
            let biometricButton = BiometricWithText(frame: .zero)
            biometricButton.textLabel.text = NativeLobbyLoginModule.titleForBiometricAuth
            view.addSubview(biometricButton)
            view.addSubviewToEdges(biometricButton)
            biometricButton.addTarget(self,
                                      action: #selector(self.performFingerprintLogin),
                                      for: .touchUpInside)
        }
```

Not a huge issue, but can be worked around for now and perhaps the do in line with the case ... : does not help I guess.

Thank you in advance for your help :).

Kind Regards,

Goffredo

**Comment by nicklockwood** [comment]
Thanks for reporting - I'll get it fixed.

**Comment by nicklockwood** [comment]
@Panajev I'm not sure what the right behavior is here. The code looks nice formatted as you have it, but what should happen if you put code inside the case statement after the end of the `do {}` block? It would either need to align with the case statement itself (which wouldn't correctly reflect the nesting), e.g.:

```swift
switch foo {
    case .bar: do {
        print("bar")
    }
    print("still in bar")
    default:
        break
}
```

Or it would be misaligned with the closing `}`, which would also seem weird:

```swift
switch foo {
    case .bar: do {
        print("bar")
    }
        print("still in bar")
    default:
        break
}
```

The natural solution would be to indent the `do {}` body an extra level, like this:

```swift
switch foo {
    case .bar: do {
            print("bar")
        }
        print("still in bar")
    default:
        break
}
```

But I'm guessing you'd prefer it didn't do that. Any thoughts on this?

**Comment by Panajev** [comment]
Hello @nicklockwood 

I think that perhaps neither of the options above is preferrable (although out of the three the third one might be best), I think do should fall to the next line:
```swift
switch foo {
    case .bar: 
        do {
            print("bar")
        }
        print("still in bar")
    default:
        break
}
```

What do you think?

**Comment by nicklockwood** [comment]
@Panajev I agree that looks better, but it seems out of scope for the `indent` rule. I'll add an issue for it.

**Comment by nicklockwood** [comment]
@Panajev fixed in 0.47.3

**Comment by Panajev** [comment]
Thanks!
